docs/installation/est/Installing_EST_pkispawn.md

```diff
@@ -0,0 +1,180 @@
+# EST installation using `pkispawn`
+
+After the prerequisite in [Installing
+EST](Installing_EST.md), it is possible to installa **EST**.
+
+## REALM user DB
+
+
+**EST** subsystem has its own realm authtnetication with a separate
+user DB. Out of the box *LDAP*, *PostreSQL* and file based DB are
```

Comment:
I jumped directly from "Using `pkispawn` [here](./Installing_EST_pkispawn.md)", I don't see any mention for an ldap instance to be created.  Did I miss a step in the jump?

Reply:
It is in the subsection.  Should I move it?

Reply:
@fmarco76 which subsection?

Reply:
I was thinking the one below from line 14. But reading again your comment maybe you want the DS installation steps, not just the how the ldap tree should be populated. Should I include the step to create the instance? (And the same for postegresql?)

Reply:
before you can do line 14 and on, you want to mention that a DS instance needs to be created.  I think there already is a page for that which you can link to.  See an example from the 2nd paragraph of Overview here : https://github.com/dogtagpki/pki/blob/master/docs/installation/ca/Installing_CA.md